### ISSUE TITLE
`previousComparator must be a LocalFileComparator on non-web, got Instance of '_TrivialWebGoldenComparator'` error

### ISSUE THREAD

**Issue by albemala** [adil192/golden_screenshot#3]
I'm getting the following error on macOS, using the latest version 2.2.1:

```
The following assertion was thrown running a test:
previousComparator must be a LocalFileComparator on non-web, got Instance of
'_TrivialWebGoldenComparator'
'package:golden_screenshot/src/_screenshot_comparator_io.dart':
Failed assertion: line 10 pos 16: 'previousComparator is LocalFileComparator'

When the exception was thrown, this was the stack:
#2      new ScreenshotComparator (package:golden_screenshot/src/_screenshot_comparator_io.dart:10:16)
#3      ScreenshotTester.useScreenshotComparator (package:golden_screenshot/src/screenshot_tester.dart:110:24)
#4      ScreenshotTester.expectScreenshot (package:golden_screenshot/src/screenshot_tester.dart:130:5)
```

I have downgraded the package to version 2.1 and it works fine, so it looks like a regression in the latest version.

P.S. Thank you for making this package!

**Comment by adil192** [comment]
This should be fixed with `golden_screenshot: ^2.2.2`!

**Comment by albemala** [comment]
Wow, that was a quick fix! I confirm it is fixed now, thank you so much
